package.json

```diff
@@ -45,7 +45,7 @@
     "clone": "2.1.1",
     "coveralls": "3.0.0",
     "dependency-graph": "0.7.0",
-    "firebase-tools": "5.1.1",
+    "firebase-tools": "6.1.1",
```

Comment:
There should generally be an accompanying yarn.lock change any time you update a dependency in a package.json file.  Can you run "yarn" from the root and see if yarn.lock gets updated (and if so, add it to this PR)?

Reply:
Oops. Reverted, since I'm not relying on firebase cli to download and start emulators. No need to update the version here.

Eventually we will leverage firebase cli to manage the download and startup of emulators once we have addressed discussion in this [doc](https://docs.google.com/document/d/1k_nmMPL9LRFzxk9yJceNYgHcxgiMeIaAc07YfKUgtqk/edit).